test/cli/repo.js

```diff
@@ -16,4 +16,16 @@ describe('repo', () => runOnAndOff((thing) => {
     const out = await ipfs('repo version')
     expect(out).to.eql(`${repoVersion}\n`)
   })
+
+  it('get repo stats', async () => {
+    const stats = await ipfs('repo stat')
+    expect(stats).to.match(/^RepoSize:\s+\d+$/gm)
+    expect(stats).to.match(/^StorageMax:\s+\d+$/gm)
+  })
+
+  it('get human readable repo stats', async () => {
+    const stats = await ipfs('repo stat --human')
+
+    expect(stats).to.match(/\s.?B$/gm)
```

Comment:
Can this regex be more specific?  At the moment it'll match ` °B`

Something similar to the other test, so:

```
expect(stats).to.match(/^RepoSize:\s+[\d\.]+\s[TGMK]?B$/gm)
expect(stats).to.match(/^StorageMax:\s+[\d\.]+\s[TGMK]?B$/gm)
```

So it'll match:

```
RepoSize: 5 B
RepoSize: 5.5 B
RepoSize: 5.5 KB
RepoSize: 5.5 MB
..etc
```